.github/workflows/ci.yaml

```
@@ -0,0 +1,83 @@
+
+          echo "Setting up K Framework..."
+          export PATH=$PATH:$(pwd)/kframework/bin
+
+      # Download and Build KEVM
+      - name: Install KEVM
+        env:
+            REPOSITORY_TOKEN: ${{ secrets.REPOSITORY_TOKEN }}
+        run: |
+          echo "Downloading KEVM..."
+          curl -H "Authorization: token $REPOSITORY_TOKEN" -L -o evm.tar.gz https://github.com/runtimeverification/evm-semantics/archive/refs/tags/v1.0.753.tar.gz
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)